modules/calibration/data/dev_kit/cancard_params/canbus.conf

```diff
@@ -0,0 +1,4 @@
+--flagfile=/apollo/modules/common/data/global_flagfile.txt
+--canbus_conf_file=/apollo/modules/canbus/conf/canbus_conf.pb.txt
+--noenable_chassis_detail_pub
```

Comment:
Suggest enable_chassis_detail_pub for dkit, because we always suggest developer open the chassi_detail in our 
guide spec.